src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -160,21 +160,13 @@ public static string ExtractUri(string uri, AccessibilityNodeInfo addressNode, B
             uri = browser.GetUriFunction(addressNode.Text)?.Trim();
             if (uri != null && uri.Contains("."))
             {
-                if (!uri.Contains("://") && !uri.Contains(" "))
+                if (Uri.TryCreate(uri, UriKind.Absolute, out var uri2))
                 {
-                    uri = string.Concat("http://", uri);
+                    return uri;
                 }
-                else if (Build.VERSION.SdkInt <= BuildVersionCodes.KitkatWatch)
+                if (Uri.TryCreate("http://" + uri, UriKind.Absolute, out var uri3))
                 {
-                    var parts = uri.Split(new string[] { ". " }, StringSplitOptions.None);
-                    if (parts.Length > 1)
-                    {
-                        var urlPart = parts.FirstOrDefault(p => p.StartsWith("http"));
-                        if (urlPart != null)
-                        {
-                            uri = urlPart.Trim();
-                        }
-                    }
```

Comment:
This seems like something that was more custom. Are we sure it's not needed? I cannot recall.

Reply:
It's behind a check for `Build.VERSION.SdkInt <= BuildVersionCodes.KitkatWatch` which we no longer support.